Package.swift

```diff
@@ -61,7 +61,10 @@ let package = Package(
             resources: [.copy("MBXInfo.plist")]),
         .target(
             name: "CTestHelper",
-            dependencies: ["MapboxMobileEvents"]),
+            dependencies: [
+                "MapboxMobileEvents",
+                "MapboxCoreNavigation",
```

Comment:
Interesting that the build was passing without this change, but I agree with adding it as a dependency.

Reply:
Yeah, no idea why wouldn't it build...